kubernetes/platform-services/values.yaml

```diff
@@ -245,10 +245,10 @@ attribute-service:
   resources:
     requests:
       cpu: "0.25"
-      memory: "512Mi"
+      memory: "128Mi"
```

Comment:
Took away some extra resources from other low-utilization services so the new service shouldn't require a larger overall footprint.